com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/ResumeProcessAction.java

```diff
@@ -39,8 +40,11 @@ public void executeActualProcessAction(String userId, String superProcessInstanc
     private void triggerProcessInstance(String userId, String processId) {
         List<Execution> executionsAtReceiveTask = flowableFacade.findExecutionsAtReceiveTask(processId);
         if (!executionsAtReceiveTask.isEmpty()) {
-            executionsAtReceiveTask.stream()
-                .forEach(execution -> flowableFacade.trigger(userId, execution.getId()));
+            executionsAtReceiveTask
+                .forEach(execution -> {
+                    flowableFacade.setProcessVariable(execution.getId(), Constants.VAR_USER, userId);
```

Comment:
Can we please extract the two behaviour lines in a separate method and use method ref ?